server/serverMethods.js

```diff
@@ -93,12 +93,16 @@ function ServerMethods(aLogLevel, aModules) {
       var templatingSecret = config.get(C.TEMPLATING_SECRET);
       var apiKey = config.get(C.OPENTOK_API_KEY);
       var apiSecret = config.get(C.OPENTOK_API_SECRET);
+      var precallApiKey = config.get(C.OPENTOK_PRECALL_API_KEY) || config.get(C.OPENTOK_API_KEY);
```

Comment:
Hmm this looks good to me, but I'm thinking that someone might accidentally add the API KEY TO `C.OPENTOK_PRECALL_API_KEY` and API SECRET to `C.OPENTOK_API_SECRET` which would be a miss match.

Maybe I'm overthinking it, let me know what you think.

Reply:
I am not concerned about this. The intent here is that the current behavior (using the main API key and secret for the precall test) continues if you do not set the precall API key/secret config values (which are undocumented).

Reply:
sounds good 😄